azdev/utilities/command.py

```diff
@@ -33,7 +33,7 @@ def call(command, **kwargs):
     """
     return subprocess.call(
```

Comment:
I think it's better to get rid of these old APIs and stick with `subprocess.run`, like what we did in https://github.com/Azure/azure-cli/pull/29745.